linkedList/homework1.php

```diff
@@ -0,0 +1,50 @@
+<?php
+require_once('linkedList.php');
+
+$list = new LinkedList(); 
+$list->insert(10); 
+$list->insert(4); 
+$list->insert(1); 
+$list->insert(2);
+$list->insert(5);
+$list->insert(2);
+$list->insert(3);
+
+$list->deleteFirst(10);
+
+$list->visit();
+echo "<br>";
+
+class LinkedList2 extends LinkedList 
+{
+    public function deleteAll($data) {
+        if ($this->head == null){
+            echo "List is empty.";
+            return;
+        }
+        if ($this->head->getData() == $data){
+            $this->head = $this->head->getNext();
```

Comment:
Please check test with the case delete duplicated on first head

```
$list2 = new LinkedList2(); 
$list2->insert(2); 
$list2->insert(4); 
$list2->insert(1); 
$list2->insert(2);
$list2->insert(5);
$list2->insert(2);
$list2->insert(3);
$list2->deleteAll(2);
$list2->visit()
```

Reply:
My former code o works as your feedback, working differently from what homework question requires.

I changed the sentences of function `deleteAll` into the one working accurately 
(changed into 2 `if` sentence & 1 `while` sentence.)
 & I added another test code at the end as you proposed.